orchestration-shared/src/test/java/uk/gov/di/orchestration/shared/helpers/ConstructUriHelperTest.java

```diff
@@ -16,11 +17,11 @@ class ConstructUriHelperTest {
 
     @Test
     void shouldBuildUriWhenOnlyBaseUrlIsPresent() {
-        var baseUrl = "https://GOV.UK";
+        var baseUrl = "https://GOV.UK/";
```

Comment:
if so, how confident are you that nothing depended on the previous behaviour?

Reply:
Raised this concern as not 100% sure we cover checking the Uri in each usage of the helper. Would feel it would be good to manually test this somehow. Not sure if it could be tested in sandpit but maybe staging and running through different journeys ?

Reply:
I think if you've looked at the usages and you're not too worried that's ok